test/docfx.Test/build/TocTest.cs

```diff
@@ -37,11 +37,11 @@ public static class TocTest
         public static void FindTocRelativePath(string[] tocFiles, string file, string expectedTocPath)
```

Comment:
shall we use spec test so we don't need to change test case when refactor source code 😆

Reply:
><, spec test is not always suitable, especially for very special purpose.

---
In reply to: [188491520](https://github.com/dotnet/docfx/pull/2773#discussion_r188491520) [](ancestors = 188491520)